[BUG]  unexpected token: > in statement [SELECT SUM(motherclubfee+previousclubcommission)

**Describe the bug**

The following error appears in the log:

``` 
 [Error]   JDBCAdapter: createPreparedStatement : java.sql.SQLSyntaxErrorException: unexpected token: > in statement [SELECT SUM(motherclubfee+previousclubcommission) FROM TRANSFER WHERE date=>? AND dete<?]
Statement: SELECT SUM(motherclubfee+previousclubcommission) FROM TRANSFER WHERE date=>? AND dete<?
java.sql.SQLSyntaxErrorException: unexpected token: > in statement [SELECT SUM(motherclubfee+previousclubcommission) FROM TRANSFER WHERE date=>? AND dete<?]
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCPreparedStatement.<init>(Unknown Source)
	at org.hsqldb.jdbc.JDBCConnection.prepareStatement(Unknown Source)
	at core.db.JDBCAdapter.createPreparedStatement(JDBCAdapter.java:63)
	at core.db.DBManager$PreparedStatementBuilder.getStatement(DBManager.java:2508)
	at core.db.TransferTable.getSumTransferCommissions(TransferTable.java:211)
	at core.db.DBManager.getSumTransferCommissions(DBManager.java:1565)
	at module.transfer.PlayerTransfer.downloadMissingTransferCommissions(PlayerTransfer.java:250)
	at core.file.xml.ConvertXml2Hrf.createHrf(ConvertXml2Hrf.java:165)
	at core.net.OnlineWorker.getHrf(OnlineWorker.java:71)
	at core.net.DownloadDialog.startDownload(DownloadDialog.java:305)
	at core.net.DownloadDialog.actionPerformed(DownloadDialog.java:87)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2313)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:297)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6626)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3389)
	at java.desktop/java.awt.Component.processEvent(Component.java:6391)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5001)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:775)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:747)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:744)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: org.hsqldb.HsqlException: unexpected token: >
	at org.hsqldb.error.Error.parseError(Unknown Source)
	at org.hsqldb.ParserBase.unexpectedToken(Unknown Source)
	at org.hsqldb.ParserBase.checkIsIdentifier(Unknown Source)
	at org.hsqldb.ParserDQL.readColumnOrFunctionExpression(Unknown Source)
	at org.hsqldb.ParserDQL.XreadSimpleValueExpressionPrimary(Unknown Source)
	at org.hsqldb.ParserDQL.XreadAllTypesValueExpressionPrimary(Unknown Source)
Caused by: org.hsqldb.HsqlException: unexpected token: >

	at org.hsqldb.ParserDQL.XreadAllTypesPrimary(Unknown Source)
	at org.hsqldb.ParserDQL.XreadAllTypesFactor(Unknown Source)
	at org.hsqldb.ParserDQL.XreadAllTypesTerm(Unknown Source)
	at org.hsqldb.ParserDQL.XreadAllTypesCommonValueExpression(Unknown Source)
	at org.hsqldb.ParserDQL.XreadRowOrCommonValueExpression(Unknown Source)
	at org.hsqldb.ParserDQL.XreadRowValuePredicand(Unknown Source)
	at org.hsqldb.ParserDQL.XreadPredicateRightPart(Unknown Source)
	at org.hsqldb.ParserDQL.XreadBooleanPrimaryOrNull(Unknown Source)
	at org.hsqldb.ParserDQL.XreadBooleanTestOrNull(Unknown Source)
	at org.hsqldb.ParserDQL.XreadBooleanFactorOrNull(Unknown Source)
	at org.hsqldb.ParserDQL.XreadBooleanTermOrNull(Unknown Source)
	at org.hsqldb.ParserDQL.XreadBooleanValueExpression(Unknown Source)
	at org.hsqldb.ParserDQL.readWhereGroupHaving(Unknown Source)
	at org.hsqldb.ParserDQL.XreadTableExpression(Unknown Source)
	at org.hsqldb.ParserDQL.XreadQuerySpecification(Unknown Source)
	at org.hsqldb.ParserDQL.XreadSimpleTable(Unknown Source)
	at org.hsqldb.ParserDQL.XreadQueryPrimary(Unknown Source)
	at org.hsqldb.ParserDQL.XreadQueryTerm(Unknown Source)
	at org.hsqldb.ParserDQL.XreadQueryExpressionBody(Unknown Source)
	at org.hsqldb.ParserDQL.XreadQueryExpression(Unknown Source)
	at org.hsqldb.ParserDQL.compileCursorSpecification(Unknown Source)
	at org.hsqldb.ParserCommand.compilePart(Unknown Source)
	at org.hsqldb.ParserCommand.compileStatement(Unknown Source)
	at org.hsqldb.Session.compileStatement(Unknown Source)
	at org.hsqldb.StatementManager.compile(Unknown Source)
	at org.hsqldb.Session.execute(Unknown Source)
	... 47 more
```

**To Reproduce**
Steps to reproduce the behavior:
1. Checkout latest code from master
2. Run HO
3. Download data from CHPP
4. Look at the logs

**Screenshots**
N/A

**Platform information:**
 - OS: Linux
 - Version HEAD in master 

**Additional context**
N/A

--- a/src/main/java/core/db/TransferTable.java
+++ b/src/main/java/core/db/TransferTable.java
@@ -203,7 +203,7 @@ public class TransferTable extends AbstractTable {
         return load(PlayerTransfer.class, this.adapter.executePreparedQuery(statement, teamId));
     }
 
-    DBManager.PreparedStatementBuilder getSumTransferCommissionsStatementBuilder = new DBManager.PreparedStatementBuilder("SELECT SUM(motherclubfee+previousclubcommission) FROM " + TABLENAME + " WHERE date=>? AND dete<?");
+    DBManager.PreparedStatementBuilder getSumTransferCommissionsStatementBuilder = new DBManager.PreparedStatementBuilder("SELECT SUM(motherclubfee+previousclubcommission) FROM " + TABLENAME + " WHERE date>=? AND date<?");
 
     public int getSumTransferCommissions(HODateTime startOfWeek) {
         var from = startOfWeek.toDbTimestamp();

--- a/src/main/resources/release_notes.md
+++ b/src/main/resources/release_notes.md
@@ -7,6 +7,7 @@
 ## [Detailed Changelog](https://github.com/ho-dev/HattrickOrganizer/issues?q=milestone%3A8.0)
 
 ### Database
+* fix `unexpected token` issue with Transfer table query (#1965)
 
 ### Squad
 * fix length of owner notes in players' database table (#1816)

